src/SceneKit/SCNNode.cs

```diff
@@ -112,5 +112,18 @@ public bool IsAnimationPaused (string key)
 
 			return isPaused;
 		}
+
+#if !XAMCORE_4_0
+		// SCNNodePredicate is defined as:
+		// 	delegate bool SCNNodePredicate (SCNNode node, out bool stop);
+		// but the actual objective-c definition of the block is
+		// 	void (^)(SCNNode *child, BOOL *stop)
+		//
+		[Obsolete ("Use the overload that takes a SCNNodeHandler instead")]
+		public virtual void EnumerateChildNodes (SCNNodePredicate predicate)
+		{
+			// FIXME: Should we throw here or error out in the Obsolete?
```

Comment:
it looks like it could use a `SCNNodeHandler` internally that calls the predicate, which is what would be the less likely to break existing code